src/v2/view-builder/views/password/EnrollAuthenticatorPasswordView.js

```diff
@@ -46,9 +46,11 @@ const Body = BaseForm.extend({
   },
 
   getPasswordPolicy () {
-    // This will be overridden by password expired and password will expire soon
-    // scenarios since the policies could be different for those.
-    return this.options.appState.get('currentAuthenticator').settings;
+    // This will be overridden by following scenario since the policies could be different for those.
+    // - password reset (`ReEnrollAuthenticatorPasswordView.js`)
+    //
+    const relatesToObject = this.options.currentViewState.relatesTo;
```

Comment:
`nit:` what do you think about this?
```
const { value: passwordPolicy } = this.options.currentViewState.relatesTo;
return passwordPolicy && passwordPolicy.settings;
```
Improves readability.

Reply:
doesn't seem improving readability to me though. 
I read as: `relatesTo has a property 'value' and rename it to passwordPolicy. return 'passwordPolicy.settings' as long as 'passwordPolicy' not null`

v.s other way (I guess I get used this already)

`return this.options.currentViewState.relatesTo.value.settings` safely (aka as long as it exists.)